.github/workflows/release-publish.yaml

```
@@ -3,16 +3,18 @@ on:
   push:
     tags:
       # These tags should be protected, remember to enable the rule:
-      # https://github.com/canonical/craft-cli/settings/tag_protection
+      # https://github.com/canonical/starbase/settings/tag_protection
```

```suggestion
      # https://github.com/canonical/craft-cli/settings/tag_protection
```

This should point to the repo's settings